Python/bytecodes.c

```diff
@@ -3761,13 +3757,8 @@ dummy_func(
     ;
 }
 
-// Families go below this point //
+// Future families go below this point //
```

Comment:
What does "future families" mean?

Reply:
These are currently ignored by the parser because they occur after the `// END BYTECODES //` marker. I plan to move them into the main section of the file as I convert families to non-legacy format.